lib/stdlib/test/gen_event_SUITE.erl

```diff
@@ -1344,21 +1344,27 @@ format_log_1(_Config) ->
 
     Depth = 10,
     ok = application:set_env(kernel, error_logger_format_depth, Depth),
-    Limited = [1,2,3,4,5,6,7,8,9,'...'],
+    Limited = [1,2,3,4,5,6,7,8,'...'],
+    LastMsg = ["Last msg" | Term],
+    State   = ["State" | Term],
+    Reason  = ["Reason" | Term],
+    LastMsgLimited = ["Last msg" | Limited],
+    StateLimited   = ["State" | Limited],
+    ReasonLimited  = ["Reason" | Limited]
     {F2,A2} = gen_event:format_log(#{label=>{gen_event,terminate},
                                      handler=>Handler,
                                      name=>Name,
-                                     last_message=>Term,
-                                     state=>Term,
-                                     reason=>Term}),
+                                     last_message=>LastMsg,
+                                     state=>State,
+                                     reason=>Reason}),
     FExpected2 = "** gen_event handler ~tP crashed.\n"
         "** Was installed in ~tP\n"
         "** Last event was: ~tP\n"
         "** When handler state == ~tP\n"
         "** Reason == ~tP\n",
     ct:log("F2: ~ts~nA2: ~tp", [F2,A2]),
     FExpected2 = F2,
-    [Handler,Depth,Name,Depth,Limited,Depth,Limited,Depth,Limited,Depth] = A2,
+[Handler,Depth,Name,Depth,LastMsgLimited,Depth,StateLimited,Depth,ReasonLimited,Depth] = A2,
 
     {WF2,WA2} = gen_event:format_log(Warning),
     WFExpected2 = "** Undefined handle_info in ~p\n"
```

Comment:
```suggestion
    WFExpected2 = "** Undefined handle_info in ~p\n"
       [?MODULE,WarningLimited,Depth] = WA2,
```

Reply:
@bucko909  The test case was a little less forward than expected. Now it will work as expected if we change row 1368
to the suggestion above! But as it was not part of the original suggestion I seem not to be able to fix it properly via the
GitHub GUI. Could you please make that last adjustment?

Reply:
Never mind found another way to fix it myself!